redfish-core/lib/power_subsystem.hpp

```diff
@@ -1,20 +1,78 @@
 #pragma once
 
 #include "app.hpp"
+#include "dbus_utility.hpp"
+#include "error_messages.hpp"
 #include "logging.hpp"
 #include "query.hpp"
 #include "registries/privilege_registry.hpp"
 #include "utils/chassis_utils.hpp"
+#include "utils/dbus_utils.hpp"
 
 #include <boost/url/format.hpp>
+#include <sdbusplus/asio/property.hpp>
+#include <sdbusplus/unpack_properties.hpp>
 
+#include <cstdint>
 #include <memory>
 #include <optional>
 #include <string>
 
 namespace redfish
 {
 
+inline void getPowerSubsystemAllocation(
+    const std::shared_ptr<bmcweb::AsyncResp>& asyncResp)
+{
+    sdbusplus::asio::getAllProperties(
+        *crow::connections::systemBus, "xyz.openbmc_project.Settings",
+        "/xyz/openbmc_project/control/host0/power_cap",
+        "xyz.openbmc_project.Control.Power.Cap",
+        [asyncResp](const boost::system::error_code& ec,
```

Comment:
#include <boost/system/error_code.hpp>

Reply:
Good catch I missed that include.  It builds OK, so I think given that it is downstream and testing has been done I'm going to leave as is.  However if I have to make any other changes I'll fix that too.